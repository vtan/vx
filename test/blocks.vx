(
  (let a 0)
  (
    (let tmp 1)
    (set a (+ a tmp))
  )
  (let b a)
  (
    (let tmp 0x10)
    (set b (+ b tmp))
  )
  (let c b)
  (let i 1)
  (while i (
    (let tmp 0x100)
    (set c (+ c tmp))
    (set i (- i 1))
  ))
)
