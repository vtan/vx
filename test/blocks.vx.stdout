00000000: 0000 0000 0000 0000 1101 0000 0000 0000  ................
00000010: 1100 0000 0000 0000 0100 0000 0000 0000  ................
